adagrams/game.py

```diff
@@ -1,11 +1,104 @@
 def draw_letters():
-    pass
+    import random
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+    draw_pool = []
+    for letter, frequency in LETTER_POOL.items():
+        for i in range(frequency):
+          draw_pool.append(letter)
+    hand = []
+    for i in range(10): # wish I had known about .sample before submitting!
```

Comment:
Sample is a handy method, but it's often more valuable to re-implement things by hand to practice thinking about how to breakdown and approach the problem than to use a single library function to solve the problem for us.

Not every language has the same built-in functions, so we should be able to implement this same functionality ourselves.